modules/tensor_mechanics/src/materials/ComputePlaneFiniteStrain.C

```diff
@@ -42,6 +44,20 @@ ComputePlaneFiniteStrain::ComputePlaneFiniteStrain(const InputParameters & param
 {
   if (_out_of_plane_strain_coupled && _scalar_out_of_plane_strain_coupled)
     mooseError("Must define only one of out_of_plane_strain or scalar_out_of_plane_strain");
+
+  if (coupledScalarComponents("scalar_out_of_plane_strain") == 0)
```

Comment:
What happens here if 'scalar_out_of_plane_strain' is not given in the input file?  Does this line need to be protected with 'if (_scalar_out_of_plane_strain_coupled)'?

Reply:
I believe this should still work even if scalar_out_of_plane_strain is not provided as the call coupledScalarComponents returns the following:

unsigned int
ScalarCoupleable::coupledScalarComponents(const std::string & var_name)
{
  return _coupled_scalar_vars[var_name].size();
}

That should return 0 if scalar_out_of_plane_strain is not provided.  All of the plane strain tests that use ComputePlaneFiniteStrain should be verifying this behavior as they do not specify scalar_out_of_plane_strain in their input files.